bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/update/instructions.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
+<update:update-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:update="https://openhab.org/schemas/update-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/update-description/v1.0.0 https://openhab.org/schemas/update-description-1.0.0.xsd">
+
+	<thing-type uid="netatmo:presence">
+
+		<instruction-set targetVersion="1">
+			<add-channel id="siren">
+				<type>netatmo:siren-status-rw</type>
+			</add-channel>
+			<update-channel id="status" groupIds="siren">
+				<type>netatmo:siren-status</type>
```

Comment:
I was rather expecting a tag `itemType` and not `type`.

Reply:
The is no `itemType` tag, only `type`. I looked at some examples in the `openhab-addons` repo, it was done this way also.

Reply:
You're right, I just checked https://www.openhab.org/schemas/update-description-1.0.0.xsd and only type, label, description and tags are allowed.
By updating the type, the itemType will be updated when PR openhab/openhab-core#3576 will be merged (I tested that with your meteoalerte PR).